etc/dev/docker-compose.yml

```diff
@@ -8,6 +8,10 @@ services:
   php:
     ports:
       - "2323:22"
+    volumes:
+      - ./etc/dev/php/docker-xdebug.ini:/etc/php7/conf.d/docker-php-ext-xdebug.ini
+    environment:
+      - PHP_IDE_CONFIG=serverName=es.local
```

Comment:
Remove this ones

Reply:
Environment value is required to debug scripts when you run inside PHP container:

```bash
make s=php sh
# Inside container
php bin/console app:create-user foo@foo.com foo@foo.com
```

Otherwise you'll see this error message on debug tab

![image](https://user-images.githubusercontent.com/17051512/83528401-59d9c280-a4e9-11ea-9809-2a5e99d1f332.png)

Reply:
Oh didn't knew. I always run the command as PHP scripts in phpstorm. Keep it then